src/components/elements/carousel/carousel-style.ts

```diff
@@ -1,9 +1,27 @@
 import React from 'react'
 import styled, { css } from 'styled-components'
+import type { FlattenSimpleInterpolation } from 'styled-components'
 import { Flex } from 'components/containers'
 import device from 'themes/device'
 import { ReactComponent as Chevron } from 'images/svg/careers/carousel-chevron.svg'
 
+type StyledButtonWrapperProps = {
```

Comment:
based on the naming convention of deriv-com it should be `StyledButtonWrapperPropsType`, is that correct?

Reply:
No @ali-hosseini-fs as I checked they are using one of these 2 conventions: `ComponentNameProps` || `ComponentNameTypes`